Client/Frontend/Toolbar+URLBar/TabLocationView.swift

```diff
@@ -110,15 +114,23 @@ class TabLocationView: UIView, FeatureFlaggable {
         shareButton.contentHorizontalAlignment = .center
         shareButton.accessibilityIdentifier = AccessibilityIdentifiers.Toolbar.shareButton
         shareButton.accessibilityLabel = .TabLocationShareAccessibilityLabel
+        shareButton.showsLargeContentViewer = true
+        shareButton.largeContentImage = .templateImageNamed(ImageIdentifiers.share)
+        shareButton.largeContentTitle = .TabLocationShareButtonAccessibilityLabel
```

Comment:
I reused an existing accessibility label. Should I create a separate localized string for the `largeContentTitle`?